tests/js/spec/components/group/inboxReason.spec.jsx

```diff
@@ -44,7 +49,12 @@ describe('InboxReason', () => {
         }}
       />
     );
-    const tooltip = mountWithTheme(wrapper.find('Tooltip').prop('title'));
-    expect(tooltip.text()).toContain('Affected 10 user(s)');
+    const tag = screen.getByText('Unignored');
+    userEvent.hover(tag);
+
+    // Text is split up because of translations
+    expect(await screen.findByText('Affected')).toBeInTheDocument();
+    expect(await screen.findByText('10')).toBeInTheDocument();
+    expect(await screen.findByText('user(s)')).toBeInTheDocument();
```

Comment:
these probably don't all need to be awaits, but its also a little gross to check for all 3

Reply:
oops, that is copy paste derpiness.